blockchain-introduction.md

```diff
@@ -0,0 +1,35 @@
+# Blockchain introduction
+
+Cosmos already has a [great intro on blockchains](https://cosmos.network/intro) so you can start there
+This page is more about how Agoric sees and uses blockchains
+
+## A blockchain is like a machine
+
+> a blockchain is a machine which state is made of consensus, not of silicon
+
+(source?)
+
+![A Blockchain, validators each with its ledger copy](./images/blockchain.svg)
+
+A blockchain can be described as a **digital ledger** maintained by a set of **validators**. Each square above is one of these **validators** (a physical machine) and each has its own copy of the **ledger**
+
+The blockchain network acts like a single virtual machine
+
+
+## Communicating with this machine
+
+### Solo node
+
+[Cosmic SwingSet](https://github.com/Agoric/cosmic-swingset) allows to build a **solo node**. This solo node can communicate with the rest of the blockchain network and add transactions to it.
+
+![A blockchain with solo nodes at the periphery](./images/blockchain-solo-nodes.svg)
+
+(TODO: Do they have their own copy of the ledger as well?)
```

Comment:
I failed to find the info and this would have a consequence on the drawing

Reply:
Each instance making up a chain has to have their own separate copy of the state.

Reply:
It's not clear to me whether this includes solo nodes

Does each solo node have its own copy of the ledger?